src/controllers/cohort.js

```diff
@@ -30,3 +30,12 @@ export const create = async (req, res) => {
     return sendMessageResponse(res, 500, ERR.UNABLE_TO_CREATE_COHORT)
   }
 }
+
+export const getAll = async (req, res) => {
+  try {
+    const foundCohorts = await getAllCohorts()
+    return sendDataResponse(res, 200, { cohorts: foundCohorts })
+  } catch (e) {
+    return sendMessageResponse(res, 500, { error: ERR.UNABLE_TO_GET_COHORTS })
+  }
```

Comment:
Let's add a `console.error(ERR.UNABLE_TO_GET_COHORTS, e)` in this catch to log out the actual error message on the server. I include both the user-friendly error message and the exact error message so it's easier for devs to find